config/.env

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    secret: "s3cr3tH"
```

Comment:
@felipehfs You can't expose secret keys on the public repo. Here you have to replace by node env, or exclude this file and leave for the environment take care of this.